c-sharp-apps-nisim-rachum/Common/ProcessManager.cs

```diff
@@ -0,0 +1,51 @@
+﻿using c_sharp_apps_nisim_rachum.BankApp;
+using c_sharp_apps_nisim_rachum.DraftApp;
+using c_sharp_apps_nisim_rachum.SportApp;
+using c_sharp_apps_nisim_rachum.TransportationApp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace c_sharp_apps_nisim_rachum.Common
+{
+    internal class ProcessManager
+    {
+        public static void RunMainProcess ()
+        {
+            Console.WriteLine("Choose one options: ");
+            Console.WriteLine("1 – Bank App | 2 – Sport App | 3 – Transportation App | 4 – Draft App | 0- Exit");
+
```

Comment:
I suggest you for not writing duplicate code to delete those 2 lines above and just write below

`int choose = 1;`

Then take the 2 lines that you have put at 43-47 and put them in the start of the while.